### DISCUSSION TITLE
de-opkomst-van-ai

### DISCUSSION THREAD

**Discussion by jayka45** [fdnd/docs.fdnd.nl#270] [Reflectie]
Leuk om hierover te praten, in dit research artikel staat dat studenten juist creatiever worden wanneer ze AI gebruiken. 
[Onderzoeksartikel "How does generative artificial intelligence impact student creativity?"](https://www.sciencedirect.com/science/article/pii/S2713374523000316)

Quantitative Results
A series of two tail, paired-sample t-tests were conducted to compare people's responses with and without the help of AI. Results indicated that AI significantly increased people's creativity in all four aspects. However, as mentioned above, there are other factors that might lead to increased creativity. Therefore, the quantitative results are exploratory in nature.
When respondents brainstormed ideas about the use of paperclip with AI (M = 7.53, SD = 3.63), their originality score was significantly higher than when they did it without AI (M = 5.47, SD = 2.87), t (74) = 5.18, p < .001.
Similarly, when it comes to the flexibility of ideas for uses of a paperclip, people generated more flexible use of a paperclip when they used AI (M = 7.55, SD = 1.88) than when they did not used AI (M = 6.01, SD = 1.78), t (74) = 6.68, p < .001.
When using AI, there were also more ideas in total (fluency with AI: M = 10.84, SD = 3.67; without AI M = 8.00, SD = 2.77, t (74) = 7.34, p < .001).

Maar leuk om hier verder over te filosoferen, niet zo belangrijk voor de MTR ;-) Wel in het kader van rekenmachine of navigatiegebruik en het effect op rekenen en kaartlezen etc. Ik ben ook docent en geef AI lessen aan marketeers.

**Comment by ju5tu5** [comment]
Thanks! Ik heb de discussie aangescherpt en de referentie naar jouw artikel opgenomen. Wij sparren hier als team graag over door, we spelen met het idee om een AI-guild op te richten (evenals een accessibility-guild).
